src/github/graphql.ts

```diff
@@ -612,10 +631,11 @@ export interface ContributionsCollection {
 export interface UserResponse {
 	user: {
 		login: string;
-		avatarUrl?: string;
-		bio?: string;
-		company?: string;
-		location?: string;
+		avatarUrl: string;
+		email: string;
+		bio: string;
+		company: string;
```

Comment:
The `?` on these is correct. A user doesn't have to have these set.

```suggestion
		avatarUrl?: string;
		email?: string;
		bio?: string;
		company?: string;
```

Reply:
A user doesn't have to set them, but that will make them null/empty, not undefined/missing, in the API response. From the GraphQL schema, these fields will always return some value. It's one of those cases where null vs undefined mean very different things. Unless I'm reading the schema wrong or there have been schema changes?

![image](https://user-images.githubusercontent.com/675956/217588629-0d71d65d-1804-4818-9366-d107839e8fd7.png)

![image](https://user-images.githubusercontent.com/675956/217588590-5e20248c-3fb3-4d7d-87fe-cbbae30a7337.png)

![image](https://user-images.githubusercontent.com/675956/217588528-0865c494-21a0-43dc-b289-e5873bda9a1f.png)

![image](https://user-images.githubusercontent.com/675956/217588428-d125f927-4e15-40f0-b4f8-2ff3e578923e.png)

Example response:
![image](https://user-images.githubusercontent.com/675956/217589376-1b81628e-a97f-4021-aab0-64d8700efc80.png)

Reply:
I'll change them to `string | null` as the schema defines them.